providers/azureblob/src/main/java/org/jclouds/azure/storage/filters/SharedKeyLiteAuthentication.java

```diff
@@ -81,27 +88,82 @@
    @Inject
    public SharedKeyLiteAuthentication(SignatureWire signatureWire,
          @org.jclouds.location.Provider Supplier<Credentials> creds, @TimeStamp Provider<String> timeStampProvider,
-         Crypto crypto, HttpUtils utils) {
+         Crypto crypto, HttpUtils utils, @Named("sasAuth") boolean sasAuthentication) {
       this.crypto = crypto;
       this.utils = utils;
       this.signatureWire = signatureWire;
+      this.storageUrl = URI.create("https://" + creds.get().identity + ".blob.core.windows.net/");
       this.creds = creds;
+      this.identity = creds.get().identity;
+      this.credential = creds.get().credential;
       this.timeStampProvider = timeStampProvider;
+      this.isSAS = sasAuthentication;
    }
-
+   
+   /** 
+   * this is an updated filter method, which decides whether the SAS or SharedKeyLite 
```

Comment:
Javadoc comments should look like this (space before *):

```
/**
 *
 */
```

Reply:
Thanks, fixed